polkadot/runtime/westend/src/weights/pallet_bags_list.rs

```diff
@@ -99,12 +105,36 @@ impl<T: frame_system::Config> pallet_bags_list::WeightInfo for WeightInfo<T> {
 	/// Proof: `VoterList::ListBags` (`max_values`: None, `max_size`: Some(82), added: 2557, mode: `MaxEncodedLen`)
 	fn put_in_front_of() -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `1889`
+		//  Measured:  `1923`
 		//  Estimated: `11506`
-		// Minimum execution time: 83_012_000 picoseconds.
-		Weight::from_parts(85_139_000, 0)
+		// Minimum execution time: 80_579_000 picoseconds.
+		Weight::from_parts(83_923_000, 0)
 			.saturating_add(Weight::from_parts(0, 11506))
-			.saturating_add(T::DbWeight::get().reads(10))
+			.saturating_add(T::DbWeight::get().reads(11))
 			.saturating_add(T::DbWeight::get().writes(6))
 	}
+	/// Storage: `VoterList::CounterForListNodes` (r:1 w:0)
+	/// Proof: `VoterList::CounterForListNodes` (`max_values`: Some(1), `max_size`: Some(4), added: 499, mode: `MaxEncodedLen`)
+	/// Storage: `VoterList::Lock` (r:1 w:0)
+	/// Proof: `VoterList::Lock` (`max_values`: Some(1), `max_size`: Some(0), added: 495, mode: `MaxEncodedLen`)
+	/// Storage: `VoterList::NextNodeAutoRebagged` (r:1 w:1)
+	/// Proof: `VoterList::NextNodeAutoRebagged` (`max_values`: Some(1), `max_size`: Some(32), added: 527, mode: `MaxEncodedLen`)
+	/// Storage: `VoterList::ListBags` (r:200 w:4)
```

Comment:
reading 200 bags for rebagging 10 nodes (as per westend configs) still sounds wrong to me. It is likely an over-pessimistic benchmark, or somewhere in your code you are doing a full iteration on all bags. 

Since being overpessimistic is okay compared to the opposite, I am okay with merging this, but it needs a follow-up investigation into this.